packages/marshal/exported.js

```diff
@@ -1 +1,2 @@
-import './src/types.js';
+// eslint-disable-next-line import/export
+export * from './src/types.js';
```

Comment:
+1 to this pattern of explicit type exports over implicit.

We shouldn't have to disable the linter to do the right thing. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/export.md says it prevents "funny business"; doesn't TypeScript cover that? I propose we consider disabling this rule in our eslint config

Reply:
@michaelfig, could we not include this `export * from './src/types.js';` in the `index.js`, so that we can get rid of the `exported.js`? I believe the only purpose it served was for making sure marhsal ambient types were loaded, which we want to get rid of, right?

Reply:
I have gone ahead and done that. It reads much better.